docs/tutorials/tutorial-1.rst

```diff
@@ -0,0 +1,175 @@
+Tutorial 1 - Sandbox
```

Comment:
- in this case the term "tutorial" is actually more accurate because the goal is to set up and run a sample hello world application, which users might not necessarily need but acts as an example workflow
- generally for task-oriented topics it's better to name titles to reflect the action/goal like "Deploying a Hello World application in the Batavia Sandbox" or something less wordy :)